.github/workflows/continuous-integration.yml

```
@@ -127,3 +127,11 @@ jobs:
           else
             echo -e "\033[PHPUnit execution skipped.\033[0m"
           fi
+      - name: "Jest"
+        run: |
+          if [[ -f "node_modules/.bin/jest" && -f "jest.config.js" && -d "tests/js" ]]; then
```

For the existing steps, we only check presence of the binary and its configuration file.
Maybe we should let plugins declare the path of their tests using `testMatch` configuration (e.g. `"testMatch": ["tests/js/**/*.js"]`).
```suggestion
          if [[ -f "node_modules/.bin/jest" && -f "jest.config.js" ]]; then
```